tests/test_nlp_soap_extraction_service.py

```
@@ -0,0 +1,115 @@
+import json
+from types import SimpleNamespace
+
+import pytest
```

Ruff (0.12.2)

`pytest` imported but unused

Remove unused import: `pytest`

(F401)